Bottom: e6bef76a588ddcd129f0d536ca6bdc1a2432d3a7
Top:    bbcc5dbabf85bad2e5b3c730eed9956202864b8d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 01:13:53 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index fa03ba7..9f445f1 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2611,7 +2611,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
